src/components/controls/ProductSelector.svelte

```diff
@@ -0,0 +1,50 @@
+<script>
+  import DownCarat from 'udgl/icons/DownCarat.svelte';
+  import MenuList from 'udgl/menu/MenuList.svelte';
+  import MenuListItem from 'udgl/menu/MenuListItem.svelte';
+  import { store } from '../../state/store';
+  import DimensionMenu from './DimensionMenu.svelte';
+  import { productKeys } from '../../config/products';
+
+  const COMPACT = true;
+  const OFFSET = 10;
+</script>
+
+<style>
+  .pull-right-edge {
+    margin-right: calc(var(--space-1h) * -1);
+    display: grid;
+    align-items: center;
+  }
+
+  .main-filter__label {
+    display: grid;
+    grid-auto-flow: column;
+    align-items: center;
+    grid-column-gap: var(--space-1h);
+    font-weight: 500;
+  }
+
+  .product-selector {
+    position: relative;
+    left: -90px; /* FIXME: this is a bad approach */
```

Comment:
Would it be possible to avoid this by wrapping the search box and product selector together in a new `div`? For example, something like:

```html
<div class="search-box-and-product-selector">
  <SearchBox />
  <ProductSelector />
</div>
```

```css
.search-box-and-product-selector {
  display: grid;
  grid-auto-flow: column;
  grid-gap: 10px;
}
```